public/app/features/dashboard/components/TransformationsEditor/TransformationPickerNg.tsx

```diff
@@ -56,18 +55,24 @@ export function TransformationPickerNg(props: TransformationPickerNgProps) {
     data,
   } = props;
 
+  // Use a callback ref to call "click" on the search input
+  // This will focus it when it's opened
+  const searchInputRef = useCallback((input: HTMLInputElement) => {
```

Comment:
Maybe dumb question, but why do it this way instead of using the `autoFocus`?

Reply:
at least in pure DOM/html `autoFocus` only works on initial page load, iirc. i dunno if React gives this prop special treatment and triggers this automagically when components mount.

Reply:
Ah, that makes sense.

Reply:
That's correct, it will only focus when it's initially rendered whereas we want to focus upon drawer open